### ISSUE TITLE
Problem with controllerMiddleware()

### ISSUE THREAD

**Issue by jordankobellarz** [garygreen/pretty-routes#16]
I am having this exception while visiting /routes:

`Call to protected method Illuminate\Routing\Route::controllerMiddleware() from context 'Illuminate\View\Engines\PhpEngine'`

This problem is ocurring because `controllerMiddleware` is a protected method and is being called out of context in `routes.blade.php` view file on line 66.

**Comment by garygreen** [comment]
What laravel version you on?

**Comment by jordankobellarz** [comment]
Laravel 5.2

**Comment by cdarken** [comment]
Broken on 5.2 since 0.0.2. Tested each version.

**Comment by cdarken** [comment]
@garygreen https://github.com/garygreen/pretty-routes/blob/master/views/routes.blade.php#L65
shoud be:

```
@if (is_callable([$route, 'controllerMiddleware']))
```

`method_exists` returns true for private or protected methods too

**Comment by garygreen** [comment]
Thanks for reporting. Fixed in 0.0.5

**Comment by cdarken** [comment]
I will comment here but if you request, I will open another issue:
Taking a look inside Laravel's Route class, the method `controllerMiddleware` appeared first in Laravel 5.2 and it was protected. In Laravel 5.3 it's public but it's also called in the method `gatherMiddleware`. I think the condition can be safely replaced with:

```
@if (is_callable([$route, 'gatherMiddleware']))
```

and the next line should be:

```
{{ implode(', ', array_map($middlewareClosure, $route->gatherMiddleware())) }}
```
